core/common/src/RealityModelDisplaySettings.ts

```diff
@@ -53,6 +76,7 @@ export interface RealityModelDisplayProps {
 
 /** Settings that control how a point cloud reality model is displayed within a [Viewport]($frontend).
  * @note This is an immutable type - to modify its properties, use [[clone]].
+ * @note EDL mode is ignored (off) if the view is not perspective (camera is off)
```

Comment:
Please put some basic description of what eye-dome lighting is and ideally a link to some online source of more detailed info.

Reply:
Is this the correct location for the definition?

Reply:
Ideally, the 7 variables that control EDL would be bundled together into a type called something like EyeDomeLightingSettings. The description of what EDL is would go on that type's documentation and the `edl` prefix would be removed from each variable's name. (Also we would spell out "weight" (singular?) instead of abbreviating to "wts" (plural?)).

But I'm not going to insist on that. If those variables remain where you've put them, then the description of EDL belongs in the documentation of PointCloudDisplaySettings.

Reply:
I agree with your suggestions, but due to retest time, etc. and it being needed ASAP, I think we should go with it as is.